src/components/dateTimePicker/__tests__/index.spec.tsx

```diff
@@ -26,24 +26,24 @@ jest.mock('../../../optionalDependencies', () => {
 });
 
 describe('DateTimePicker', () => {
-  test.each(['time', 'date'] as const)('should not invoke onChange when value is not changed - mode %s', mode => {
+  test.each(['time', 'date'] as const)('should invoke onChange when value is changed - mode %s', async mode => {
```

Comment:
Why is there an `async`?

Reply:
Missed that. Its not needed anymore after I've used the driver.